dbld/Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+def jobs = []
+
+if (!params.PLATFORMS)
+    jobs = ["centos-7", "debian-stretch", "debian-buster", "ubuntu-xenial", "ubuntu-bionic", "ubuntu-cosmic", "fedora-30"]
```

Comment:
Is there a plan for syncing this platform list with dbld/rules BUILDER_IMAGES list (for now this is a duplicated)?